src/cli/cmd.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+var path = require('path');
+var async = require('async');
+var db = require('../database');
+var plugins = require('../plugins');
+
+function parseCommands(pluginList, callback) {
+	var commands = [];
+
+	pluginList.forEach(function (pluginData) {
+		var pluginCommands = pluginData.commands || [];
+
+		pluginCommands.forEach(function (cmdData) {
+			var pluginName = pluginData.id;
+			var libraryFile = cmdData.library ? cmdData.library : pluginData.library;
+			var cmdName = pluginName.replace('nodebb-plugin-', '') + ':' + cmdData.cmd;
+			var scriptFile = path.resolve(pluginData.path, libraryFile);
+			var action = require(scriptFile)[cmdData.method];
```

Comment:
Can this be done lazily?

Reply:
Running require when the command is run, by passing a closure as the action, like so:
```js
function action(...args) {
  require(scriptFile)[cmdData.method](...args);
}
```

Reply:
Ok, can I write it with es6 syntax as your example?

Reply:
@pitaj https://github.com/NodeBB/NodeBB/pull/6901/commits/f104d0d806f5ffc75b5a04af83f835e4a42e3b25

Reply:
@bonaccorsop sure, if you want to change it to that it's fine